### PR TITLE
Added Some Resources

### PR DESCRIPTION
**Pull request by TenTraicion**
## What does this PR do?
Add resources 

### Description
3 resources

### Why is this valuable (or not)?
Resources for web developers.

### How do we know it's really free?
By visiting the attached link. It mentioned on the website also.

### For book lists, is it a book? For course lists, is it a course? etc.
A Course, A bootcamp and an Web Doc.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -602,6 +603,7 @@
 * [Bento JavaScript Learning Track](https://bento.io/topic/javascript) (Bento)
 * [Cycle.js Fundamentals](https://egghead.io/courses/cycle-js-fundamentals) - André Staltz
 * [ES6 and Typescript Tutorial](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhI0V5mE9Vu6Nm-nap8EcjV) - Codevolution, Vishwas Gopinath (YouTube)
+* [Free Javascript Course](https://www2.ironhack.com/lp/free-courses/javascript) - IronHack
```
Comment (eshellman) [inline courses/free-courses-en.md:606]: Can't tell if this is free.
Don't invent titles! Use the title given by the resource.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1111,6 +1113,7 @@
 * [Frontend UI Development with React](https://www.youtube.com/playlist?list=PL0vfts4VzfNgUUEtEjxDVfh4iocVR3qIb) - Jeff Delaney, Fireship.io (YouTube)
 * [Full Stack Foundations](https://www.udacity.com/course/full-stack-foundations--ud088) - by AWS on Udacity
 * [Full stack open](https://fullstackopen.com/en/) - University of Helsinki, Houston Inc., Terveystalo, Elisa, K-ryhmä, Unity Technologies, Konecranes
+* [Full Stack Web Developement Bootcamp](https://www.appacademy.io/course/app-academy-open) - App Academy
```
Comment (eshellman) [inline courses/free-courses-en.md:1116]: link does not go to the course. Can't use it here

Comment (TenTraicion) [reply courses/free-courses-en.md:1116]: image
@eshellman by scrolling down a bit you can get all the infos you will gain there. and also you need to signup if you want to take the bootcamp.

Comment (eshellman) [reply courses/free-courses-en.md:1116]: not good enough. remove it

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md:606]: have you taken the course?

File: courses/free-courses-en.md (hunk 2 above)
Comment (TenTraicion) [reply courses/free-courses-en.md]: @eshellman should i remove the third one also?

**Pull request merged**